txscript/sign_test.go

```diff
@@ -166,7 +178,6 @@ func TestSignTxOutput(t *testing.T) {
 				mkGetKey(map[string]addressToKey{
 					address.EncodeAddress(): {key, false},
 				}), mkGetScript(nil), nil); err != nil {
-				t.Error(err)
```

Comment:
Why did you remove the error log from here?

Reply:
by mistake, this line whole expression was really hard to understand so I de-structured it and accidentally removed the error